apacheconfig/lexer.py

```diff
@@ -4,20 +4,23 @@
 # Copyright (c) 2018-2019, Ilya Etingof <etingof@gmail.com>
 # License: https://github.com/etingof/apacheconfig/LICENSE.rst
 #
+from __future__ import unicode_literals
+
 import logging
 import re
+import six
```

Comment:
super nit: we should eventually sort imports alphabetically throughout the whole code base...

Reply:
It looks like [this flake8 plugin](https://github.com/PyCQA/flake8-import-order) can help with that. Will do in a separate PR :)